be/src/exec/olap_common.h

```diff
@@ -239,6 +244,28 @@ class ColumnValueRange {
         }
     }
 
+    void to_boundary_condition(std::vector<TCondition>& filters) {
+        for (const auto& value : _boundary_values) {
+            TCondition condition;
+            condition.__set_column_name(_column_name);
+            condition.__set_compound_type(_compound_type);
```

Comment:
`condition.compound_type` and `get_compound_type` do not seem to be used

Reply:
You are right, I've removed it.